tuning/config/acceleration_configs/attention_and_distributed_packing.py

```diff
@@ -33,3 +33,6 @@ class AttentionAndDistributedPackingConfig:
     def __post_init__(self):
         # ensure nested dataclasses initialized
         ensure_nested_dataclasses_initialized(self)
+
+    def is_padding_free(self):
```

Comment:
nit: make this a property with decorator `@property`

Reply:
done